src/_gettsim/transfers/benefit_checks/benefit_checks.py

```diff
@@ -1,67 +1,97 @@
-def wohngeld_vorrang_hh(
-    wohngeld_nach_vermög_check_m_hh: float,
-    arbeitsl_geld_2_vor_vorrang_m_bg: float,
+aggregate_by_group_benefit_checks = {
+    "wohngeld_vorrang_wthh": {
+        "source_col": "wohngeld_vorrang_bg",
+        "aggr": "any",
+    },
+    "wohngeld_kinderzuschl_vorrang_wthh": {
+        "source_col": "wohngeld_kinderzuschl_vorrang_bg",
+        "aggr": "any",
+    },
+}
+
+
+def wohngeld_vorrang_bg(
+    wohngeld_nach_vermög_check_m_bg: float,
+    arbeitsl_geld_2_regelbedarf_m_bg: float,
+    arbeitsl_geld_2_eink_m_bg: float,
 ) -> bool:
     """Check if housing benefit has priority.
 
+    Housing benefit has priority if the sum of housing benefit and income covers the
+    needs according to SGB II of the Bedarfsgemeinschaft.
+
     Parameters
     ----------
-    wohngeld_nach_vermög_check_m_hh
-        See :func:`wohngeld_nach_vermög_check_m_hh`.
-    arbeitsl_geld_2_vor_vorrang_m_bg
-        See :func:`arbeitsl_geld_2_vor_vorrang_m_bg`.
+    wohngeld_nach_vermög_check_m_bg
+        See :func:`wohngeld_nach_vermög_check_m_bg`.
+    arbeitsl_geld_2_regelbedarf_m_bg
+        See :func:`arbeitsl_geld_2_regelbedarf_m_bg`.
+    arbeitsl_geld_2_eink_m_bg
+        See :func:`arbeitsl_geld_2_eink_m_bg`.
 
     Returns
     -------
 
     """
-    # TODO(@MImmesberger): Allow for some individuals to receive Wohngeld and some
-    # individuals to receive ALG2 in the same household.
-    # https://github.com/iza-institute-of-labor-economics/gettsim/issues/710
-    return wohngeld_nach_vermög_check_m_hh >= arbeitsl_geld_2_vor_vorrang_m_bg
+    return (
+        arbeitsl_geld_2_regelbedarf_m_bg
+        <= wohngeld_nach_vermög_check_m_bg + arbeitsl_geld_2_eink_m_bg
```

Comment:
```suggestion
        <= arbeitsl_geld_2_eink_m_bg + wohngeld_nach_vermög_check_m_bg
```

Easier to read.

I'd actually prefer to change the order of the arguments, too:

```py
    arbeitsl_geld_2_regelbedarf_m_bg: float,
    arbeitsl_geld_2_eink_m_bg: float,
    wohngeld_nach_vermög_check_m_bg: float,
```

(first the need, then what the need is compared against)